configure.ac

```diff
@@ -861,7 +861,11 @@ if test "$use_hardening" != "no"; then
   esac
 
   case $host in
-    *aarch64*)
+    *aarch64-*-linux*)
+      AX_CHECK_COMPILE_FLAG([-mbranch-protection=standard], [HARDENED_CXXFLAGS="$HARDENED_CXXFLAGS -mbranch-protection=standard"])
+      AX_CHECK_LINK_FLAG([-Wl,-z,pac-plt], [HARDENED_LDFLAGS="$HARDENED_LDFLAGS -Wl,-z,pac-plt"], [], [$LDFLAG_WERROR])
```

Comment:
Could this be an issue?

https://sourceware.org/bugzilla/show_bug.cgi?id=26312

Reply:
I'll take a look, and we can backport something if relevant.